src/components/AccountMenu/AccountMenu.tsx

```diff
@@ -0,0 +1,59 @@
+import { IconChevronDown, IconUser } from '@tabler/icons-react';
+
+import { Dropdown, DropdownContent, DropdownTrigger } from './../DropdownMenu';
+
+interface AccountMenuProps {
+  userName: string;
+  loggedAsRole: string;
+  logoutText: string;
+  children?: React.ReactNode;
+  className?: string;
+}
+
+export const AccountMenu: React.FC<AccountMenuProps> = ({
+  userName,
+  loggedAsRole,
+  children,
+  className = '',
+}) => {
+  return (
+    <div className={className}>
+      <Dropdown>
+        <DropdownTrigger
+          asChild
+          className="rounded-sm bg-surface-tertiary cursor-pointer
+            transition-shadow duration-150
+            hover:shadow-[0_0_0_4px_var(--chemican-green-100)]
```

Comment:
I think you should be able to use the `ring-4 ring-interactive-focused` instead. The Figma design uses a shadow to achieve that, but that's due to a Figma technicality.